packages/notebook-extension/src/index.ts

```diff
@@ -731,6 +733,67 @@ function addCommands(
     },
     isEnabled
   });
+  commands.addCommand(CommandIDs.runInConsole, {
+    label: 'Run Selected Text or Current Line in Console',
+    execute: args => {
+      // Default to not activating the notebook (thereby putting the notebook
+      // into command mode)
+      const current = getCurrent({ activate: false, ...args });
+
+      if (current) {
```

Comment:
My preference is to have this be an early exit instead of a long if. That way, the code is less nested and imho clearer, because we can follow the main logic in the outer scope and we see this is just an early termination check. 

So this would be replaced by:

```typescript
      if (!current) {
        return Promise.resolve()
      }
```